crates/common/tedge_config/tests/test_tedge_config.rs

```diff
@@ -139,11 +148,12 @@ port = 1883
     let updated_azure_url = "OtherAzure.azure-devices.net";
     let updated_mqtt_port = Port(2345);
     let updated_mqtt_external_port = Port(3456);
-    let updated_mqtt_external_bind_address = "localhost";
+    let updated_mqtt_external_bind_address = IpAddress(std::net::IpAddr::V4(Ipv4Addr::LOCALHOST));
```

Comment:
As the import is used more than once (line 156) maybe it could be imported with the use of `use` instead?

Reply:
It can also be implemented as `IpAddress::default()`

Reply:
Yes, changed it to IpAddress::default()